app/Resources/assets/css/events.scss

```diff
@@ -103,15 +103,13 @@
     }
 
     .save-participants-form {
+        .new-participants-label label:after {
```

Comment:
Psuedoelement should be prefixed with two colons per W3C spec: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements#Syntax

Reply:
I learn something every day.

Fixed.